File: ios/CountlyTestApp-iOS/TestViewControllerModal.m
// TestViewControllerModal.m
//
// This code is provided under the MIT License.
//
// Please visit www.count.ly for more information.

#import "TestViewControllerModal.h"

@interface TestViewControllerModal ()
@end

@implementation TestViewControllerModal

- (void)viewDidLoad
{
    [super viewDidLoad];
}

- (IBAction)onClick_dismiss:(id)sender
{
    [self dismissViewControllerAnimated:YES completion:nil];
}

@end

File: ios-swift/CountlyTestApp-iOS-swift/TestViewControllerModal.swift
// TestViewControllerModal.swift
//
// This code is provided under the MIT License.
//
// Please visit www.count.ly for more information.

import UIKit

class TestViewControllerModal: UIViewController
{
    override func viewDidLoad()
    {
        super.viewDidLoad()
    }


    @IBAction func onClick_dismiss(_ sender: Any)
    {
        self.dismiss(animated:true, completion: (() -> Void)?{})
    }
}
